docs/spec_draft.md

```diff
@@ -454,6 +455,48 @@ implementation-defined.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.logistic
+
+`stablehlo.logistic(operand) -> result`
+
+### Semantics
+
+Performs element-wise logistic (sigmoid) function on `operand` tensor and
+produces a `result` tensor. For floating-point element types, it implements:
+$$logistic(x)=\frac{1}{1+e^{-x}}$$
+For complex element types, computes a complex logistic function, with corner
+cases TBD. Numeric precision is implementation-defined.
+
+### Operands
+
+| Name | Type |
+|-|-|
+| `operand` | tensor of floating-point or complex element types |
+
+### Results
+
+| Name | Type |
+|-|-|
+| `result` | tensor of floating-point or complex element types |
+
+### Constraints
+
+  * (C1) `result` must have the same type as that of `operand`.
```

Comment:
Like what we have for other ops. 

(C1) `operand` and `result` have the same type.